torchmetrics/audio/pesq.py

```diff
@@ -0,0 +1,98 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional
+
+from torch import Tensor, tensor
+
+from torchmetrics.functional.audio.pesq import pesq
+from torchmetrics.metric import Metric
+
+
+class PESQ(Metric):
+    """PESQ (Perceptual Evaluation of Speech Quality)
+
+    This is a wrapper for the pesq package [1].
+
+    Forward accepts
+
+    - ``preds``: ``shape [...,time]``
+    - ``target``: ``shape [...,time]``
+
+    Args:
+        preds:
+            shape ``[...,time]``
+        target:
+            shape ``[...,time]``
+        fs:
+            sampling frequency, should be 16000 or 8000 (Hz)
+        mode:
+            'wb' (wide-band) or 'nb' (narrow-band)
+        keep_same_device:
+            whether to move the pesq value to the device of preds
+
+    Example:
+        >>> from torchmetrics.functional.audio import pesq
+        >>> import torch
+        >>> preds = torch.randn(8000)
+        >>> target = torch.randn(8000)
+        >>> nb_pesq_val = pesq(preds, target, 8000, 'nb')
+        >>> wb_pesq_val = pesq(preds, target, 16000, 'wb')
```

Comment:
show the outputs if possible
in case it is too long use ellipsis

Reply:
In the example, two random array are used (preds, target), so I dont know the outputs...
`pesq` requires at two 2000-element array. it's hard to construct that **long meaningful numbers**
Do you any ideas about how to solve this?

Reply:
what about setting a seed in the docstring?
then you just need to run it locally and then you can just insert the value.

Reply:
OK. I will try

Reply:
done